pkg/loadbalancer/loadbalancer.go

```diff
@@ -573,7 +574,7 @@ func NewL3n4AddrFromModel(base *models.FrontendAddress) (*L3n4Addr, error) {
 func NewBackend(id BackendID, protocol L4Type, ip net.IP, portNumber uint16) *Backend {
 	lbport := NewL4Addr(protocol, portNumber)
 	b := Backend{
```

Comment:
Should you add a DefaultWeight here too?

Reply:
yeah you are right! thanks

Reply:
done